src/commands/update-wcw.ts

```diff
@@ -23,19 +24,65 @@ export async function execute(interaction: CommandInteraction<CacheType>) {
         return;
     }
 
-    await Promise.all([
-        addTopic(topic),
-        interaction.reply({
-            content: 'Topic has been added ✅',
-            ephemeral: true,
-        }),
-        logger({
-            project: 'blacc',
-            channel: 'general',
-            event: 'WCW topic added',
-            description: `Called by ${member.id}`,
-            icon: '🟢',
-            notify: true,
-        }),
-    ]);
+    await interaction.reply({
+        content: `Are you sure you want to add the topic: ${topic}?`,
+        components: ConfirmationButton,
+    });
+
+    const filter = (i: any) => i.customId === 'confirm' || i.customId === 'cancel';
```

Comment:
Can we remove the anys here? @DarrylBrooks97

Reply:
Just typed it, removing just through errors @gt-codes